lib/libzpool/kernel.c

```diff
@@ -848,6 +848,8 @@ dprintf_find_string(const char *string)
 	return (0);
 }
 
+char olaf[] = "mmp.c";
```

Comment:
Yes.  Just a way to make a temporary change easy to find.  My name doesn't appear in code bases often.